packages/site/src/main.css

```diff
@@ -1,5 +1,11 @@
 :root {
-  --sideBarWidth: 250px;
+  --sideBarWidth: 200px;
+}
+
+@media screen and (min-width: 1024px) {
+  :root {
+    --sideBarWidth: 235px;
```

Comment:
It's a bit smaller than before but still avoids wrapping on long hook and guide titles:
![image](https://github.com/user-attachments/assets/2f32be84-ed26-4fb4-87c2-c6546569ba9a)

Reply:
(Left: before, right: after)